vars/mavenBuild.groovy

```diff
@@ -53,7 +53,16 @@ def call(body) {
         stage('Package') {
             try {
                 sh "${mvnCmd} package"
+
+                //TODO: Build mba files that fortify can use to perform scans, so we won't have compiler errors in the fortify build
```

Comment:
Is this still a valid TODO?

Reply:
@jasonluck nope, it's not. It can be deleted.